.github/workflows/pr-build.yml

```
@@ -1,18 +1,24 @@
-name: Build (Nightly)
+name: PR Build (Preview)
+run-name: '#${{ github.event.pull_request.number }} - ${{ github.event.pull_request.title }} @${{ github.event.pull_request.head.ref }}:${{ github.event.pull_request.head.sha }}'
 
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - hotfix-*
-      - test-pr-*
+  pull_request:
+    types: [opened, synchronize, reopened]
     paths-ignore:
       - '**/*.md'
       - README.md
-  pull_request:
-    types: [opened, synchronize, reopened]
 
 jobs:
+  context:
+    runs-on: ubuntu-22.04
```

It seems this job is nothing special. We can use ubuntu-latest safely here.
```suggestion
    runs-on: ubuntu-latest
```